tests/src/Functional/NewsSearchTest.php

```diff
@@ -92,11 +92,12 @@ public function testNewsSearch() {
     $this->assertSession()->pageTextContains('Test News Article');
 
     // Defaults to be on 'news' path page.
-    $this->drupalGet('news/2021/test-news-article');
+    $year = date("Y");
```

Comment:
Rather than using the current date it might make more sense to also check that the _created_ date is being used for the article path date, rather than the current date. Something like:-

```
diff --git a/tests/src/Functional/NewsSearchTest.php b/tests/src/Functional/NewsSearchTest.php
index 0d31e9e..097f3e0 100644
--- a/tests/src/Functional/NewsSearchTest.php
+++ b/tests/src/Functional/NewsSearchTest.php
@@ -74,6 +74,8 @@ class NewsSearchTest extends BrowserTestBase {
       'type' => 'localgov_news_article',
       'status' => NodeInterface::PUBLISHED,
       'localgov_newsroom' => ['target_id' => $newsroom->id()],
+      'created' => 1590962400,
+      // 2020-06-01 00:00:00.
     ]);
 
     $this->drupalLogout();
@@ -92,12 +94,11 @@ class NewsSearchTest extends BrowserTestBase {
     $this->assertSession()->pageTextContains('Test News Article');
 
     // Defaults to be on 'news' path page.
-    $year = date("Y");
-    $this->drupalGet("news/$year/test-news-article");
+    $this->drupalGet("news/2020/test-news-article");
     $this->submitForm(['edit-search-api-fulltext' => 'dogma'], 'Apply');
     $this->assertSession()->pageTextContains('Test News Article');
 
-    $this->drupalGet("news/$year/test-news-article");
+    $this->drupalGet("news/2020/test-news-article");
     $this->submitForm(['edit-search-api-fulltext' => 'xyzzy'], 'Apply');
     $this->assertSession()->pageTextNotContains('Test News Article');
   }
```

Reply:
I've done this correctly and added it into a PR that is requesting to update this PR ;-) #74